horace_core/sqw/@sqw/calculate_qw_pixels2.m

```diff
@@ -79,13 +84,15 @@
     kf = ki;
 end
 
-qw = cell(1, 4);
-qw(1:3) = calculate_q (ki, kf, detdcn(:, idet), spec_to_rlu(:, :, irun));
+qw = cell(4, 1);
+qw(1:3) = calculate_q(ki, kf, detdcn(:, idet), spec_to_rlu(:, :, irun));
 qw{4} = eps_diff;
+qw = cat(2, qw{:})';
```

Comment:
be good to comment what these lines do as they look rather obscure

Reply:
```suggestion
% Join cell array into 4xN mat
qw = cat(2, qw{:})';
```